CRM/Core/EntityTokens.php

```diff
@@ -16,7 +16,14 @@
 use Civi\Token\TokenProcessor;
 
 /**
- * Class CRM_Core_EntityTokens
+ * Generic base-class which loads tokens via APIv4.
+ *
+ * To write a subclass:
+ *
+ * - (MUST) Implement getEntityName() and getApiEntityName()
+ * - (MUST) Implement getApiTokens()
```

Comment:
I think this should be 'May' implement getApiTokens. - We don't need to implement this if we just use the metadata